### DISCUSSION TITLE
How to get and follow the newer release..mine seems to be a old install.

### DISCUSSION THREAD

**Discussion by nealhamiltonjr** [mealie-recipes/mealie#1342] [General]
I followed the instructions here for the sql light version. 

https://hay-kot.github.io/mealie/documentation/getting-started/install/

But my install was missing the backup section like you show on how to backup up the files...and then I noticed it says the version is. 0.5.6.

I then followed the update instructions and restarted the container but it still shows the old version. Just to be clear I installed this today and with the docker compose file provided it pulled that old version. I did a manual docker pull and it says it's the latest but still the old version shows and there's no backup options. 

The ui says this version is the latest: v1.0.0beta-2

For all setups using Docker the updating process looks something like this

    Stop the container using docker-compose down
    Pull the latest image using docker-compose pull
    Start the container again using docker-compose up -d

**Comment by Bentipa** [comment]
Check the installation page for the nightly (currently beta2) build: https://nightly.mealie.io/documentation/getting-started/installation/installation-checklist/

Be sure to replace v1.0.0-beta-1 with v1.0.0beta-2 (yeah no extra -) in the docker-compose files.

**Comment by nealhamiltonjr** [comment]
Yep, I finally figured it out; that was hard to find and should be easier.
There should also be a tag so you don't have to change the beta1 to 2 and
so on. Like a beta-latest or something. What I'd like to know is it seems
to suggest that from one beta to another might not be compatible with the
database schemes so your entered data might not work as we move from beta
to beta to then prod. Is this true? I really like this product and don't
mind testing and helping nor the bugs but I'd also like to use the product
which is why I found it. I've got about 2k recipes I'd like to enter but
don't want to waste time. A lot would be manual entry. This software could
bring the home kitchen into this century...so much potential.
